requirements/build.txt

```diff
@@ -0,0 +1,11 @@
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    pip-compile requirements/build.in
+#
+-e file:///home/seanh/Projects/via3  # via -r requirements/build.in
```

Comment:
Nope!

Reply:
Fixed using the workaround here: https://github.com/jazzband/pip-tools/issues/204#issuecomment-550051424